blockchain/evm/evm.go

```diff
@@ -1,20 +1,18 @@
-package blockchain
+package evm
 
 import (
-	"bytes"
 	"encoding/hex"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"math/big"
-	"net/http"
-	"net/url"
 
-	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/external-initiator/blockchain/common"
+
+	eth "github.com/ethereum/go-ethereum/common"
```

Comment:
After going down a rabbit hole online, I've learned that dot-imports (which I had planned for our internal `common` package) is not a good thing to use, and should only ever be used in tests.

Instead, the GETH common package is imported as `eth`.